### ISSUE TITLE
Cannot modify iron-form request property from the attribute

### ISSUE THREAD

**Issue by jfcieslak** [PolymerElements/iron-form#124]
### Description
I am trying to get an iron-form response as text, not as json.
To do so, I need to change `request.handleAs` property of `iron-form` element. I can do that easily in adjacent script tag. However I would expect that this property could be modified via html attribute change.
But none of the below works:

`<form is="iron-form" method="get" action="/index.html" handle-as="text">`
`<form is="iron-form" method="get" action="/index.html" handleas="text">`
`<form is="iron-form" method="get" action="/index.html" request='{"handleAs" : "text"}'>`

The response always comes back as json.
Is there a way to change the `request.handleAs` attribute through html?

**Comment by notwaldorf** [comment]
No, that isn't possible right now, but would probably be a good addition.

Given that the form isn't setting this anywhere, you could just set it once, after creating the form, with something like
```
  form.request.handleAs = 'text';
```

Or use the presubmit hook to update this before firing the request:
```
form.addEventListener('iron-form-presubmit', function() {
      this.request.handleAs = 'text';
    });
```
